float64 multipleX
float64 multipleY

---

float64 valX
float64 valY